tamr_client/instance/backup.py

```diff
@@ -0,0 +1,94 @@
+from tamr_client import response
+from tamr_client._types import Instance, Session, URL
+from tamr_client.exception import TamrClientException
+
+
+class AlreadyCanceled(TamrClientException):
+    """Raised when attempting to cancel a backup that has already failed or been canceled.
+    """
+
+    pass
+
+
+class NotFound(TamrClientException):
+    """Raised when referencing a backup that does not exist on the server.
+    """
+
+    pass
+
+
+def get(session: Session, instance: Instance) -> list:
+    """Get information on Tamr backups.
+
+    Args:
+        session: Tamr session
+        instance: Tamr instance
+
+    Returns:
+        A list of Tamr backups
+    """
+    url = URL(instance=instance, path='backups')
+    r = session.get(str(url))
+    if r.status_code == 404:
+        raise NotFound(str(url))
+    return response.successful(r).json()
+
+
+def get_by_id(session: Session, instance: Instance, backup_id: str) -> dict:
```

Comment:
To mirror the `from_resource_id` functions for projects/datasets/etc, this could be `from_id`.

Reply:
if we have `backups()` to get all backups shouldn't it be something like `backup_by_id()`. It seems inconsistent to have the word backup in one function name but not the other.

Reply:
This naming definitely will be influenced by the decision on the function that retrieves all backups.  See the comment there about avoiding repetition in the names.

Reply:
Should be named `from_resource_id`, since `id` could be confused with the universal ID (e.g. https://github.com/Datatamer/tamr-client/pull/438/files#diff-362c9b0ea842fd1de6fde077c5610098R10 )

Reply:
updated to `from_resource_id` for now. Although I kind of like `by_resource_id` better when fetching from Tamr